pkg/doc/util/time.go

```diff
@@ -0,0 +1,140 @@
+package util
+
+import (
+	"time"
+)
+
+// TimeWithTrailingZeroMsec overrides marshalling of time.Time. It keeps a format of initial unmarshalling
+// in case when date has zero a fractional second (e.g. ".000").
+// For example, time.Time marshals 2018-03-15T00:00:00.000Z to 2018-03-15T00:00:00Z
+// while TimeWithTrailingZeroMsec marshals to the initial 2018-03-15T00:00:00.000Z value.
+type TimeWithTrailingZeroMsec struct {
```

Comment:
Can also be called 'TimeExt', 'TimeX' etc, I am fine with current name too

Reply:
good idea!